scripts/index/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+<meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+<meta content="utf-8" http-equiv="encoding">
+
+<link rel="stylesheet" type="text/css" href="html/browser.css">
+
+<script type="text/javascript" src="html/jquery-3.3.1.min.js"></script>
+<script type="text/javascript" src="html/artifacts.js"></script>
+<script type="text/javascript" src="html/browser.js"></script>
+
+<title id="title"></title>
+</head>
+
+<body>
+<h1 id="header"></h1>
```

Comment:
We should uses classes instead of IDs for `header`, `names`, `versions`, and `listings` as they are used multiple times in this page.

Reply:
They really need to be uniquely identifiable though. I could give them unique names if that's clearer. I just always reference them as e.g. `#binaries #header` in CSS or JS code though. Otherwise it would look like `#binaries #binaries-header`.

Reply:
What if I were to use "name" like in this answer:
https://stackoverflow.com/questions/9680037/how-do-i-select-an-element-with-its-name-attribute-in-jquery

Is that better?

I don't want to use "class" because I normally think of classes being used to apply style, which is not the case here.

Reply:
Yes, `name` would be a better fit.